gamemode/gametypes/gametype_base.lua

```diff
@@ -105,6 +106,60 @@ function GAMETYPE:IsPlayerEnemy(ply1, ply2)
+    self:AddSetting("walkspeed",{Category = "SERVER", NiceName = "#GM_WALKSPEED", value_type = "string", value = 150, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Walk speed" })
+    self:AddSetting("normspeed",{Category = "SERVER", NiceName = "#GM_NORMSPEED", value_type = "string", value = 190, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Walk speed" })
+    self:AddSetting("sprintspeed",{Category = "SERVER", NiceName = "#GM_SPRINTSPEED", value_type = "string", value = 320, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Sprint speed" })
+    self:AddSetting("connect_timeout",{Category = "SERVER", NiceName = "#GM_CONNECTTIMEOUT", value_type = "string", value = 120, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Time required before player is considered to time out." })
+    self:AddSetting("playercollision",{Category = "SERVER", NiceName = "#GM_PLAYERCOLLISION", value_type = "bool", value = 1, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Enables or disables collisions between players." })
+    self:AddSetting("friendlyfire",{Category = "SERVER", NiceName = "#GM_FRIENDLYFIRE", value_type = "bool", value = 0, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Enables friendly fire, only works if player collisions enabled." })
+    self:AddSetting("weapon_strip_force",{Category = "SERVER", NiceName = "#GM_PREVENTITEMMOVE", value_type = "bool", value = 1, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Prevents players from moving weapons and items from by shooting." })
+    self:AddSetting("limit_default_ammo",{Category = "SERVER", NiceName = "#GM_DEFAMMO", value_type = "bool", value = 1, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "If enabled weapons default ammo will use the sk_* settings for max limit." })
+    self:AddSetting("allow_auto_jump",{Category = "SERVER", NiceName = "#GM_AUTOJUMP", value_type = "bool", value = 150, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Allow automatically jumping if on ground" })
+    self:AddSetting("max_respawn_timeout",{Category = "SERVER", NiceName = "#GM_RESPAWNTIME", value_type = "string", value = 20, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Time before player can respawn" })
+    self:AddSetting("map_restart_timeout",{Category = "SERVER", NiceName = "#GM_RESTARTTIME", value_type = "string", value = 20, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY, FCVAR_REPLICATED), info = "Time before a new round starts when all players are dead" })
+    self:AddSetting("instance_id",{Category = "SERVER", NiceName = "#GM_INSTANCEID", value_type = "string", value = 1, flags = bit.bor(0, FCVAR_ARCHIVE, FCVAR_NOTIFY), info = "Allows to assign a unique instance id to support multiple srcds instances at once from the same directory." })
```

Comment:
This setting isn't supposed to change during gameplay, shouldn't be part of gametype.